src/lando/pushlog/tests/conftest.py

```diff
@@ -0,0 +1,126 @@
+from datetime import datetime
+
+import pytest
+
+from lando.main.models import Repo
+from lando.main.scm.commit import Commit as SCMCommit
+from lando.pushlog.models import Commit, File, Push, Tag
+
+
+@pytest.fixture
+def make_repo():
+    def repo_factory(seqno: int) -> Repo:
+        "Create a non-descript repository with a sequence number in the test DB."
+        # Model.objects.create() creates _and saves_ the object
```

Comment:
nit: This is self-evident, not sure we have to add a comment about it.

Reply:
I wasn't aware of it before I discovered it (:

But yes, this is pretty standard, and documented, Django.